utils/python/CIME/SystemTests/nck.py

```diff
@@ -41,21 +40,21 @@ def build(self):
                         self._case.set_value("ROOTPE_%s"%comp, "%s"%int(rootpe/2))
             self._case.flush()
 
-
             run_cmd("case.setup -clean -testmode")
```

Comment:
I had to undo some of your changes to this file in order to get identical behavior in both one-phase and two-phase builds. Might be worth thinking about those changes and if we need to bring them back in some form.